fuchsia.py

```diff
@@ -1794,15 +1794,15 @@ def import_matrix_mathematica(f):
     from a file-like object.
     """
     data =  f.read()
-    data = data.translate(None, ' \n')
+    data = data.translate(str.maketrans('','',' \n'))
```

Comment:
This doesn't work on Python2, doesn't it?

Reply:
Not sure to be honest. Note that most of the code has already been updated to python3 in the latest commit. Maybe branch for python2 vs python3?

Reply:
Nah, no branches needed so far, I think `data.replace(" ", "").replace("\n", ...` should do the same thing as this does, and not break compatibility. Sage has only recently (and finally!) became py3 compatible, so keeping py2 compatibility for a while longer makes sense. I think it is easy enough to support both in our case.